### PR TITLE
Add notes for the scheduler setup

### PR DESCRIPTION
**Pull request by AIC-BV**
```
&> /dev/null        // notation works
>> /dev/null 2>&1   // notation doesn't work
```
Thanks @mjauvin

### CODE REVIEW

**Comment by AIC-BV** [comment]
BOTH work.
Hosting provider replied, they made a change in the permissions, so I guess both work but there was a wrong setup on their end.
They also updated my php path to the correct path

**Comment by AIC-BV** [comment]
Perhaps better like the latest commit?

**Review by LukeTowers — APPROVED** [review]

**Comment by LukeTowers** [comment]
@bennothommo can you fix the linter?

File: setup/installation.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ If you intend to use scheduled tasks, or install plugins that use scheduled task
 * * * * * php /path/to/artisan schedule:run >> /dev/null 2>&1
 ```
 
-Be sure to replace `/path/to/artisan` with the absolute path to the `artisan` file in the root directory of your Winter installation. This cron task will call the command scheduler every minute, to which Winter will evaluate any scheduled tasks and run the tasks that are due for execution.
+Be sure to replace `php` with the absolute path to the `php` file and also replace `/path/to/artisan` with the absolute path to the `artisan` file in the root directory of your Winter installation. This cron task will call the command scheduler every minute, to which Winter will evaluate any scheduled tasks and run the tasks that are due for execution.
```
Comment (LukeTowers) [inline setup/installation.md]:
```suggestion
This cron task will call the command scheduler every minute, to which Winter will evaluate any scheduled tasks and run the tasks that are due for execution.

> **NOTE**: If `php` is not on your PATH, then make sure you provide the absolute path to the version of PHP you wish to run the scheduler with. `/path/to/artisan` should also reflect the absolute path to the `artisan` file in the root directory of your Winter installation.
```

File: setup/installation.md [hunk 2]
```diff
@@ -95,7 +95,9 @@ If you intend to use scheduled tasks, or install plugins that use scheduled task
 * * * * * php /path/to/artisan schedule:run >> /dev/null 2>&1
 ```
 
-Be sure to replace `/path/to/artisan` with the absolute path to the `artisan` file in the root directory of your Winter installation. This cron task will call the command scheduler every minute, to which Winter will evaluate any scheduled tasks and run the tasks that are due for execution.
+This cron task will call the command scheduler every minute, to which Winter will evaluate any scheduled tasks and run the tasks that are due for execution.
+
+> **NOTE**: If `php` is not on your PATH, then make sure you provide the absolute path to the version of PHP you wish to run the scheduler with. `/path/to/artisan` should also reflect the absolute path to the `artisan` file in the root directory of your Winter installation.
```
Comment (LukeTowers) [inline setup/installation.md]:
```suggestion
```

**Pull request merged**